src/main/scala/viper/gobra/frontend/Config.scala

```diff
@@ -89,6 +89,31 @@ object MCE {
   object Enabled extends Mode
 }
 
+object MoreJoins {
+  sealed trait Mode {
+    // Option number used by Viper, as described in
+    // https://github.com/viperproject/silicon/pull/823
+    def viperValue: Int
+  }
+  object Disabled extends Mode {
+    override val viperValue = 0
+  }
+  object Impure extends Mode {
+    override val viperValue = 1
+  }
+  object All extends Mode {
+    override val viperValue = 2
+  }
+
+  def merge(m1: Mode, m2: Mode): Mode = {
+    (m1, m2) match  {
+      case (All, _) | (_, All) => All
+      case (Impure, _) | (_, Impure) => Impure
+      case _ => Disabled
+    }
+  }
+}
+
```

Comment:
For some reason, Github is showing the diff for old commits that were merged in master already. Maybe we need to merge master here

Reply:
I fixed it now